packages/app/src/cli/services/app-logs/utils.ts

```diff
@@ -239,16 +242,17 @@ export const subscribeToAppLogs = async (
 }
 
 export function prettyPrintJsonIfPossible(json: unknown) {
-  try {
-    if (typeof json === 'string') {
-      const jsonObject = JSON.parse(json)
-      return JSON.stringify(jsonObject, null, 2)
-    } else if (typeof json === 'object' && json !== null) {
-      return JSON.stringify(json, null, 2)
-    } else {
+  if (typeof json === 'string') {
```

Comment:
Will look into this, its related to Nicks comment above. Right now with PR we dont decode that part of the payload, so to display the JSON string better I added that. But thinking, it probably makes more sense to decode the inner payload here. Will look into it.

Reply:
Would be great if you could add a return type so it's easier to grok what this function is trying to return! 

My main question is why you removed the other cases from within the try/catch block instead of just changing the throw to return json. 

Additionally, why are we changing this to return json instead of throwing an error?

Reply:
> why are we changing this to return json instead of throwing an error?

Reverted this, but originally was because this values the we needed to use this method on is from a metafield, and it could technically just be a string, and we dont want to error in those cases and just show whatever the value was. That way partners can use it to debug any issue related to the input query variables.